xml/ftp_banners.xml

```diff
@@ -808,4 +808,17 @@ against these patterns to fingerprint FTP servers.
     <param pos="1" name="host.name"/>
     <param pos="2" name="service.version"/>
   </fingerprint>
+  <fingerprint pattern="^Welcome to the (?:Cisco )?(?:Tele[Pp]resence) ([a-zA-Z\s]*?) ((?:MSE )?\d+), version (\d+.\d+\(\d+.\d+\)).*?">
```

Comment:
OK -- please use ```flags="REG_ICASE"``` instead and file a Nexpose ticket to get that fixed.  While we (surprisingly) don't use that syntax right now, I can definitely see it being used at some point in the future.